discovery-provider/src/utils/helpers.py

```diff
@@ -179,6 +179,7 @@ def update_ipfs_peers_from_user_endpoint(update_task, cnode_url_list):
 
 latest_block_redis_key = 'latest_block_from_chain'
 latest_block_hash_redis_key = 'latest_blockhash_from_chain'
+most_recent_indexed_block_redis_key = 'most_recently_indexed_block_from_db'
```

Comment:
for later, these should all get moved out of here into constants or something else